src/instrument.h

```diff
@@ -45,7 +48,13 @@ class Instrument : public Engine {
                                           jint class_data_len, const u8* class_data,
                                           jint* new_class_data_len, u8** new_class_data);
 
-    static void JNICALL recordSample(JNIEnv* jni, jobject unused);
+    static void JNICALL recordEntry(JNIEnv* jni, jobject unused);
+    static void JNICALL recordExit(JNIEnv* jni, jobject unused, jlong);
 };
 
+u16 updateCurrentFrame(int32_t& current_frame_old, int32_t& current_frame_new,
+                       u16 offset_delta_old, const u16* relocation_table);
+u8 parameterSlots(const char* method_sig);
+u16 instructionBytes(const u8* code, u16 index);
```

Comment:
These are not part of Instrumentation interface.
If these declarations are only needed for tests, move them to the test sources.

Reply:
Right, I didn't think about it: 994d64c1f10752105b302b923d077e144d11d3ca